homework/src/App.js

```diff
@@ -39,20 +52,20 @@ class App {
       const repo = this.repos[index];
       const contributors = await repo.fetchContributors();
 
+
       const container = document.getElementById('container');
       // Erase previously generated inner HTML from the container div
       container.innerHTML = '';
 
-      const leftDiv = Util.createAndAppend('div', container);
-      const rightDiv = Util.createAndAppend('div', container);
-
-      const contributorList = Util.createAndAppend('ul', rightDiv);
+      const leftDiv = Util.createAndAppend('div', container, { id: 'div1' });
+      const rightDiv = Util.createAndAppend('div', container, { id: 'div2' });
```

Comment:
This `rightDiv` should actually be a list (`ul`). Using a semantically appropriate element helps to make your app ARIA-compliant. In the same spirit, each contributor should be rendered as an `li`.